static/bidder-info/optidigital.yaml

```diff
@@ -0,0 +1,23 @@
+endpoint: "https://pbs.optidigital.com/bidder/openrtb2"
+endpointCompression: gzip
+maintainer:
+  email: "prebid@optidigital.com"
+gvlVendorID: 915
+openrtb:
+  version: 2.6
+capabilities:
+  app:
+    mediaTypes:
+      - banner
+  site:
+    mediaTypes:
+      - banner
+  dooh:
+    mediaTypes:
+      - banner
+userSync:
+  supports:
+    - iframe
+  iframe:
+    url: "https://scripts.opti-digital.com/js/presyncs2s.html?endpoint=optidigital&gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&redir={{.RedirectURL}}"
```

Comment:
Confirm usersync endpoint makes call to setuid endpoint
<img width="655" alt="image" src="https://github.com/user-attachments/assets/a128541e-ea5a-4fbb-80ad-f9907bbfe825" />